src/systems/animation.js

```diff
@@ -0,0 +1,31 @@
+var registerSystem = require('../core/system').registerSystem;
+var THREE = require('../lib/three');
+
+var camera = null;
+var scene = null;
+
+/**
+ * Animation system.
+ *
+ * Controls library-level animation events agnostic to implementations.
+ *
+ */
+module.exports.System = registerSystem('animation', {
+  init: function () {
+    this.sceneEl.addEventListener('camera-ready', function () {
+      scene = this.sceneEl.object3D;
+      camera = this.sceneEl.camera;
+    }, true);
+  },
+
+  /**
+   * Update animation mixers and shaders.
+   *
+   */
+  tick: function () {
+    if (scene && camera) {
+      THREE.glTFAnimator.update();

```

Comment:
I would create a ```gltf-model``` system specific to your component (there's a 1:1 relationship) Systems are global API/State available to a component type. For instance, The geometry system has a cache that the all geometry components can use.